Sources/App/Views/PackageController/Builds/BuildShow+Model.swift

```diff
@@ -15,29 +15,28 @@
 extension BuildShow {
 
     struct Model {
-        var packageName: String
+        var packageName: String?
         var repositoryName: String
         var repositoryOwner: String
         var buildInfo: BuildInfo
         var versionId: Version.Id
 
         init?(result: BuildController.BuildResult, logs: String?) {
             guard
-                let packageName = result.version.packageName,
                 let repositoryOwner = result.repository.owner,
                 let repositoryName = result.repository.name,
                 let buildInfo = BuildInfo(build: result.build, logs: logs),
                 let versionId = result.version.id
             else { return nil }
             self.init(buildInfo: buildInfo,
-                      packageName: packageName,
+                      packageName: result.version.packageName,
                       repositoryOwner: repositoryOwner,
                       repositoryName: repositoryName,
                       versionId: versionId)
         }
 
         internal init(buildInfo: BuildInfo,
-                      packageName: String,
+                      packageName: String?,
```

Comment:
To fix it, we could keep the `packageName` field in this initialiser as non-optional, and deal with the substitution in one place on line 32 above.

This is certainly a cleaner implementation, but also doesn't feel perfect.

Reply:
I'd do the same here: keep it `String` and substitute in `repositoryOwner`.